hangman.c

```diff
@@ -0,0 +1,221 @@
+/*Rumeet Goradia - Hangman*/
```

Comment:
You should have commented way more! Your code compiled with lots of warnings and didn't run properly. This was good progress though